web/src/env.mjs

```diff
@@ -106,8 +106,6 @@ export const env = createEnv({
     // Database exports
     DB_EXPORT_PAGE_SIZE: z.number().optional(),
     // Worker
-    LANGFUSE_WORKER_HOST: z.string().optional(),
```

Comment:
*question* I assume these can be removed as we don't have any HTTP traffic left from web to worker, correct?

Reply:
@hassiebp Correct. There should never be any http invocation from web to worker. All communication goes via queues.